SKFChopperApp/Db/SKFChopper.db

```diff
@@ -937,6 +937,185 @@ record(longin, "$(P)TIME") {
+    info(archive, "VAL")
+}
+
+record(ai, "$(P)V24:FSV") {
+    field(DESC, "Peak Position FSV V24")
+    field(DTYP, "asynInt32")
+    field(INP, "@asyn($(CHOP)v24_fsv,0)INT16")
+    field(SCAN, "I/O Intr")
+    field(EGU, "um")
+    info(INTEREST, "LOW")
+    info(archive, "VAL")
+}
+
+record(ai, "$(P)W24:FSV") {
+    field(DESC, "Peak Position FSV W24")
+    field(DTYP, "asynInt32")
+    field(INP, "@asyn($(CHOP)w24_fsv,0)INT16")
+    field(SCAN, "I/O Intr")
+    field(EGU, "um")
+    info(INTEREST, "LOW")
+    info(archive, "VAL")
+}
+
+record(ai, "$(P)Z12:FSV") {
+    field(DESC, "Peak Position FSV Z12")
+    field(DTYP, "asynInt32")
+    field(INP, "@asyn($(CHOP)z12_fsv,0)INT16")
+    field(SCAN, "I/O Intr")
+    field(EGU, "um")
+    info(INTEREST, "LOW")
+    info(archive, "VAL")
+}
+
+# Peak positions calc records.
+record(calc, "$(P)V13:CALC") {
+    field(DESC, "Peak position V13 Calc")
+    field(INPA, "$(P)V13:NORM CP MSS")
+    field(INPB, "$(P)V13:FSV CP MSS")
+    field(CALC, "((A+32767)/65534)*(2*B)-B")
+    field(EGU, "um")
+}
+
+record(calc, "$(P)W13:CALC") {
+    field(DESC, "Peak position W13 Calc")
+    field(INPA, "$(P)W13:NORM CP MSS")
+    field(INPB, "$(P)W13:FSV CP MSS")
+    field(CALC, "((A+32767)/65534)*(2*B)-B")
+    field(EGU, "um")
+}
+
+record(calc, "$(P)V24:CALC") {
+    field(DESC, "Peak position V24 Calc")
+    field(INPA, "$(P)V24:NORM CP MSS")
+    field(INPB, "$(P)V24:FSV CP MSS")
+    field(CALC, "((A+32767)/65534)*(2*B)-B")
+    field(EGU, "um")
+}
+
+record(calc, "$(P)W24:CALC") {
+    field(DESC, "Peak position W24 Calc")
+    field(INPA, "$(P)W24:NORM CP MSS")
+    field(INPB, "$(P)W24:FSV CP MSS")
+    field(CALC, "((A+32767)/65534)*(2*B)-B")
+    field(EGU, "um")
+}
+
+record(calc, "$(P)Z12:CALC") {
+    field(DESC, "Peak position Z12 Calc")
+    field(INPA, "$(P)Z12:NORM CP MSS")
+    field(INPB, "$(P)Z12:FSV CP MSS")
+    field(CALC, "((A+32767)/65534)*(2*B)-B")
+    field(EGU, "um")
+}
+
+# Peak positions in engineering units.
+record(ai, "$(P)V13") {
+    field(DESC, "Peak position V13")
+    field(INP, "$(P)V13:CALC CP MSS")
+    field(EGU, "um")
+    info(INTEREST, "LOW")
+    info(archive, "VAL")
+}
+
+record(ai, "$(P)W13") {
+    field(DESC, "Peak position W13")
+    field(INP, "$(P)W13:CALC CP MSS")
+    field(EGU, "um")
+    info(INTEREST, "LOW")
+    info(archive, "VAL")
+}
+
+record(ai, "$(P)V24") {
+    field(DESC, "Peak position V24")
+    field(INP, "$(P)V24:CALC CP MSS")
+    field(EGU, "um")
+    info(INTEREST, "LOW")
+    info(archive, "VAL")
+}
+
+record(ai, "$(P)W24") {
+    field(DESC, "Peak position W24")
+    field(INP, "$(P)W24:CALC CP MSS")
+    field(EGU, "um")
+    info(INTEREST, "LOW")
+    info(archive, "VAL")
+}
+
+record(ai, "$(P)Z12") {
+    field(DESC, "Peak position Z12")
+    field(INP, "$(P)Z12:CALC CP MSS")
+    field(EGU, "um")
+    info(INTEREST, "LOW")
+    info(archive, "VAL")
+}
+
 #--------------------------------------------------
```

Comment:
These records ware very similar, could a template and substitutions file be used?